zap/src/scan.py

```diff
@@ -337,7 +355,7 @@ def main(): # pylint: disable=too-many-locals
                 s.value for s in severities))
 
             (zap_filename, session_filename) = zap_compliance_scan(
-                codedx_project, zap_port, target_url, scan_type)
+                dojo_product_name, zap_port, target_url, scan_type)
```

Comment:
Do we use CodeDx for uploading ZAP findings?

Reply:
We do, but that happens further on in scan.py. There are scans that don't have codedx entries and this allows them to be tagged in logging, it doesn't affect upload code.